packages/react-components/src/localization/locales/ar-SA/strings.json

```diff
@@ -7,7 +7,9 @@
     "mutedIconLabel": "تم كتم الصوت",
     "displayNamePlaceholder": "مشارك غير مسمى",
     "participantStateRinging": "يتم الآن الاتصال...",
-    "participantStateHold": "قيد الانتظار"
+    "participantStateHold": "قيد الانتظار",

```

Comment:
did we pull in these strings from TDBuild? We have a process for this. I would prefer to pull in TDBuild strings as a separate check in.